### PR TITLE
+ History will be properly created when adding a new person from the Edit Family screen. (Fixes #1726)

### LINKED ISSUE #1726: Creation of person from Edit Family page is not logged in history
### Description

When a new person is added from the Edit Family page, nothing is inserted into the History recording that person's data or the time it was entered. This has left us with lots of people created in our system but no record of who did it or when, which is often necessary when trying to correct bad data.
### Steps to Reproduce
1. Added a new family with two people from the Add Family screen. Each one has history entries to show when they were created and what their values were.
2. Click on Edit Family from one of these new people and choose Add Person
3. Enter new person's details and Save

**Expected behavior:** 

History is created just like it is when adding a person from the New Family page

**Actual behavior:**

No history is created.
### Versions

[RockDiagnostics-EZRS8J.txt](https://github.com/SparkDevNetwork/Rock/files/445570/RockDiagnostics-EZRS8J.txt)
[RockDiagnostics-ROCK1.txt](https://github.com/SparkDevNetwork/Rock/files/445571/RockDiagnostics-ROCK1.txt)

Rock Version: Rock McKinley 5.1
Client Culture Setting: en-US

This is happening on our local install and on the rock demo site (diagnostics attached for both).

### PR DESCRIPTION
**Pull request by chrismcgrath**
# Contributor Agreement
_Have you filled out and sent your [Spark Contributor Agreement](http://www.rockrms.com/Content/RockExternal/Misc/Contributor%20Agreement.pdf) to secretary [at] sparkdevnetwork.org?_

Yes

# Context
_What is the problem you encountered that lead to you creating this pull request?_

Investigating the creation of Person records, we would often find records that had no information in the History screen about when they were created, by whom, or what their values were at creation. We found that this was happening when people were added from the Edit Family screen.

# Goal
_What will this pull request achieve and how will this fix the problem?_

To create History records when a person is added from the Edit Family screen.

# Strategy
_How have you implemented your solution?_

The ID used to save the history was the GroupMember ID rather than the Person's ID. Fixed that. 

Also removed extraneous history that was being saved to a -1 ID. 

# Tests
_If your code is a new method or function (that doesn't need a mock database or SqlServerTypes library) and can be Xunit tested [see example](https://github.com/SparkDevNetwork/Rock/blob/develop/Rock.Tests/Rock/Lava/RockFiltersTests.cs) be sure your pull request includes the corresponding unit tests in the Rock.Tests project. In all cases *you* MUST test your code before submitting a pull request._

# Possible Implications
_What could this change potentially impact? Are there any security considerations? Where could this potentially affect backwards compatibility?_

# Screenshots
_Provide us some screenshots if your pull request either alters existing UI or provides new UI. Arrows and labels are helpful._

# Documentation
_If your change effects the UI or needs to be documented in one of the existing [user guides](http://www.rockrms.com/Learn/Documentation), please provide the brief write-up here:_

### CODE REVIEW

File: RockWeb/Blocks/Crm/PersonDetail/EditGroup.ascx.cs [hunk 1]
```diff
@@ -1132,25 +1126,14 @@ protected void btnSave_Click( object sender, EventArgs e )
                                 return;
                             }
 
-                            if ( _isFamilyGroupType )
-                            {
-                                if ( person.RecordStatusValueId != recordStatusValueID )
```
Comment (nairdo) [inline RockWeb/Blocks/Crm/PersonDetail/EditGroup.ascx.cs]: I'm not sure I understand why this code was removed.  This case occurs when adding an existing person into the edited family and when their _current/old_ record status is different than what is being set on the edit form.

File: RockWeb/Blocks/Crm/PersonDetail/EditGroup.ascx.cs [hunk 2]
```diff
@@ -1246,20 +1229,20 @@ protected void btnSave_Click( object sender, EventArgs e )
                                     }
                                 }
                             }
+
```
Comment (nairdo) [inline RockWeb/Blocks/Crm/PersonDetail/EditGroup.ascx.cs]: I'm not super familiar with this code yet and I haven't gotten to the bottom of this change yet... can you briefly describe why you moved this to happen only inside the '// existing group members' block?

File: RockWeb/Blocks/Crm/PersonDetail/EditGroup.ascx.cs (hunk 1 above)
Comment (chrismcgrath) [inline RockWeb/Blocks/Crm/PersonDetail/EditGroup.ascx.cs]: Got it, thanks for the explanation, I'll put these lines back as they were. To the best of my recollection I thought I had determined that these lines were either creating bad data or would never get hit, but I just confirmed that they do get hit in your scenario.

File: RockWeb/Blocks/Crm/PersonDetail/EditGroup.ascx.cs (hunk 2 above)
Comment (chrismcgrath) [inline RockWeb/Blocks/Crm/PersonDetail/EditGroup.ascx.cs]: With the code above removed, this code did not need to be run in all instances. I'll be putting this back as it was as well, thanks for the review.

**Comment by chrismcgrath** [comment]
I reverted the changes I made that @nairdo pointed out were incorrect. This should be ready to go now.

**Pull request merged**